src/main/java/nova/energy/Unit.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package nova.energy;
+
+import nova.core.util.Identifiable;
+import nova.core.util.registry.Registry;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * @author ExE Boss
+ */
+public final class Unit implements Identifiable {
+	private static final Registry<Unit> REGISTRY = new Registry<>();
+
+	public static final Unit METRE = getOrCreateUnit("nova:metre", "Metre", "m");
+	public static final Unit GRAM = getOrCreateUnit("nova:gram", "Gram", "g").setBasePrefix(Prefix.KILO);
+
+	public static final Unit AMPERE = getOrCreateUnit("nova:ampere", "Amp", "I");
+	public static final Unit AMP_HOUR = getOrCreateUnit("nova:amp_hour", "Amp Hour", "Ah");
+	public static final Unit VOLTAGE = getOrCreateUnit("nova:voltage", "Volt", "V");
+	public static final Unit WATT = getOrCreateUnit("nova:watt", "Watt", "W");
+	public static final Unit WATT_HOUR = getOrCreateUnit("nova:watt_hour", "Watt Hour", "Wh");
+	public static final Unit RESISTANCE = getOrCreateUnit("nova:resistance", "Ohm", "R");
+	public static final Unit CONDUCTANCE = getOrCreateUnit("nova:conductance", "Siemen", "S");
```

Comment:
Because that's how Calclavia made it in [calclavia/Electrodynamics](https://github.com/calclavia/Electrodynamics)